fbs/gen.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+flatc -c --gen-mutable --scoped-enums --gen-object-api --no-includes torchcraft.fbs
```

Comment:
Nit: Maybe add a sanity check for the flatc version here?

Reply:
```
#!/bin/bash
flatcVersionRequired='1.7.2';
flatcVersionCurrent=`flatc --version`;
if [[ $flatcVersionCurrent == *"$flatcVersionRequired"* ]]; then
  flatc -c --gen-mutable --scoped-enums --gen-object-api --no-includes torchcraft.fbs
else
  echo "The current flatbuffer headers require flatc version $flatcVersionRequired."
  echo "Current flatbuffer version: $flatcVersionCurrent"
  echo "You'll either want to get a matching copy of flatc or update the headers."
fi

```